src/components/forms/FileTypesWithFunctionalEquivalents.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {FormFieldTypes, FormTable} from './forms';
+
+export type FileTypeAndFunctionalEquivalence = {
+    type: string,
+    value: string
+}
+export type FileTypesWithFunctionalEquivalentsProps = {
+    id: string,
+    onChange: ({key, value} :{key: string, value: never}) => void,
+    defaultValue: Array<FileTypeAndFunctionalEquivalence>
+}
+
+export const FileTypesWithFunctionalEquivalents = (props: FileTypesWithFunctionalEquivalentsProps) => {
```

Comment:
I don't see where this component is used. I looked at DT-1179 and the scope appears to be wider than just adding a component so I'm not clear what work should be part of this PR.

Reply:
It's used in a new form being built.  It will make an appearance soon.